examples/api/background-transactions/example1-basic.js

```diff
@@ -24,13 +24,16 @@ var transactionName = 'myCustomTransaction'
 
 // Here is an example for the first case.
 newrelic.startBackgroundTransaction(transactionName, function handle() {
-  var transaction = newrelic.getTransaction()
+  const transaction = newrelic.getTransaction()
   doSomeWork(function cb() {
     transaction.end()
   })
 })
 
 // Function to simulate async work.
+/**
+ * @param callback
```

Comment:
this is unfortunate that this is valid jsdoc.  knowing the type is what i really find useful

Reply:
Types added :)